conf/modules/lidar_lite.xml

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="lidar_lite" dir="lidar">
```

Comment:
The teraranger range finder have be placed in a folder `range_finder`. There is also a `sonar` folder. We should probably decide if we agree on a common folder for this type of sensors.